src/factory/mod.rs

```diff
@@ -575,15 +577,54 @@ impl Factory {
         Texture::new(view, sampler.0, [width, height])
     }
 
-    fn request_texture(
+    fn load_cubemap_impl<P: AsRef<Path>>(
+        paths: &CubeMapPath<P>,
+        sampler: Sampler,
+        factory: &mut BackendFactory,
+    ) -> CubeMap<[f32; 4]> {
+        use gfx::texture as t;
+        let images = paths
+            .as_array()
+            .iter()
+            .map(|path| {
+                let format = Factory::parse_texture_format(path.as_ref());
+                let file = File::open(path).unwrap_or_else(|e| {
```

Comment:
I think this is where `expect` should be used

Reply:
Why? Expect will not use error in the message.

Reply:
Hmm, true.